cmd/autoheal/awx_job.go

```diff
@@ -102,5 +107,11 @@ func (h *Healer) launchAWXJob(connection *awx.Connection, template *awx.JobTempl
 		"Request to launch AWX job from template '%s' has been sent",
 		templateName,
 	)
+	healInitiated(
+		map[string]string{
+			"heal_type":     "awsJob",
```

Comment:
s/awsJob/awxJob/

In other parts of the code we use the terms "healing action" and "healing rule". To be consistent with that I'd suggest the following names for the labels:

```
action_type
template_name
rule_name
```

Or maybe even shorter:

```
type
template
rule
```